src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -662,7 +662,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return Nothing
                 End If
 
-                Return TextSpan.FromBounds(method.BlockStatement.Span.End, method.EndBlockStatement.SpanStart)
```

Comment:
Yes, in practice it should be equivalent because of trivia rules. I just get a little nervous when thinking about implicit/explicit line continuation, colon statement terminators, preprocessor, and all of that business. I felt like the intent would be somewhat clearly that the span should not even consider the BlockStatement. Do you feel that I should change it?